### ISSUE TITLE
JXCore with Duktape?

### ISSUE THREAD

**Issue by kzc** [jxcore/jxcore#404]
Is Duktape on the JXCore roadmap? It could prove useful in low memory or high concurrency multithreaded scenarios.

A third JS engine supported by JXCore could also bolster the case for an engine agnostic C++ native module ABI for Node.

**Comment by ktrzeciaknubisa** [comment]
Someone already mentioned it here: #329

**Comment by obastemur** [comment]
>  It could prove useful in low memory or high concurrency multithreaded scenarios.
@kzc I agree. We are planning to have it inside though but I don't know how soon we can start doing that. It's kind of a priority of actual resources problem.

**Comment by drasko** [comment]
Hi all,
any news on this integration? What is the current status?

BR,
Drasko

**Comment by drasko** [comment]
Is there anyone working on this port? Also, what is the estimated size that can be obtained?

**Comment by plika** [comment]
@drasko  ATM we are not working on it. For which target you need Duktape?

**Comment by qdk0901** [comment]
The size of jxcore binary with spidermonkey engine is bigger than 7MB, and it's very slow when launched on an openwrt board with 32MB DRAM.  Porting jxcore with light weight JS engine like jerryscript will be very great on resource limited embedded devices.

**Comment by obastemur** [comment]
I've already made some experiments with both JerryScript and Duktape. Both has cons pros. IMHO, Most importantly, both needs significant amount of contribution to things get going.
